easybuild/easyconfigs/l/likwid/likwid-4.2.0-intel-config-mk.patch

```diff
@@ -0,0 +1,49 @@
+--- likwid-likwid-4.2.orig/config.mk	2016-12-22 13:22:23.000000000 +0100
++++ likwid-likwid-4.2/config.mk	2017-06-22 10:14:28.585696000 +0200
+@@ -1,7 +1,7 @@
+ # Please have a look in INSTALL and the WIKI for details on
+ # configuration options setup steps.
+ # supported: GCC, CLANG, ICC, MIC (ICC), GCCX86 (for 32bit systems)
+-COMPILER = GCC#NO SPACE
++COMPILER = ICC#NO SPACE
```

Comment:
hmm, do we really need a patch for this?

isn't using `COMPILER=ICC` in `buildopts` sufficient?